Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM --platform=linux/amd64 registry.access.redhat.com/ubi8/ubi-minimal:8.6 AS builder
+
+RUN microdnf install -y tar gzip make which
+ENV ARCHITECTURE=x64
+# install go 1.17.8
+RUN curl -O -J https://dl.google.com/go/go1.18.4.linux-amd64.tar.gz
```

Comment:
Is there anything special about the go version 1.18.4 that is hard coded? :) I see it's 1.17.8 in bf2, and the current version is 1.20.1.

Reply:
go version depends on the official ubi image support version. I think we can update it.